src/game.cpp

```diff
@@ -8936,6 +8937,18 @@ void game::drop_in_direction( const tripoint &pnt )
     u.drop( game_menus::inv::multidrop( u ), pnt );
 }
 
+void game::drop_items_off_ledge( const tripoint &pnt, int ledge_height )
+{
+    units::mass wt_dropped = u.drop_items_off_ledge( game_menus::inv::multidrop( u ), pnt );
+
+    float damage = 10 * to_kilogram( wt_dropped ) * ledge_height;
```

Comment:
linear damage from height seems wrong. double height == double velocity == quadruple kinetic energy

Reply:
Doubling height does not double velocity.

Remember conservation of energy.

Potential energy is directly propotional to height. Doubling height doubles potential energy.
Potential energy turns into kinetic energy in 1:1 ratio (ignoring air resistance).
So doubling height doubles kinetic energy.

Reply:
Good catch. I forgot that v increases with sqrt() of d under accleration, which cancels out the v^2 for kinetic energy.